### ISSUE TITLE
Optimizer failed: Desired error not necessarily achieved due to precision loss. 

### ISSUE THREAD

**Issue by kkkmmm2017** [marcino239/pilco#2]
Optimizer failed: Desired error not necessarily achieved due to precision loss.

**Comment by iljastas** [comment]
same problem here. is the example working? does it make sense to understand the implementation and build up the own problem on this workspace?

**Comment by marcino239** [comment]
Yes, I would suggest to go to the original paper for understanding how the
algorithm works.

**Comment by shivakumar-tekumatla** [comment]
Change the optimizer type to 'Nelder-Mead' from 'bgf'. It should work.
